packages/next-server/lib/head-context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const HeadManagerContext = React.createContext()
+
+export default HeadManagerContext
```

Comment:
Wondering if it would be better to export the `<HeadManagerContext/>` from `head-manager.js` instead, imo that would make more sense and probably should not effect the bundle size 🤔

Reply:
I think this is fine tbh, however as this is an easy conversion, could you rename to `.ts` and do `export const HeadManagerContext` instead of exporting default 👍

Reply:
Sure, had to pass `null` as the default value though. Should be fine anyway